exercise3/submissions/README.md

```diff
@@ -0,0 +1,14 @@
+Exercise 3 Notes
+================
+
+*Output files are written to the output directory and have a _comp extension.
+For instance output.json is replicated as output_comp.csv in the output directory
+
+*Testing.  Testing for this is also included in the driver file.
+The output file isn't comparable to the example output for a few reasons.
+1)  The output file wasn't real generated data.  It appeated to be real data for one report that was simply replicated for all reports to give an example file.
+2)  json format isn't guaranteed to be sorted.
+
```

Comment:
To have data sorted, I use `OrderedDict` instead of `dict()`. Having data sorted is helpful for our CEP Analysts who will interact with JSON data files from time to time.